monitor.py

```diff
@@ -48,18 +53,18 @@ def get_sensor_data(self):
         and call getJson to get the values
         return
         """
-        data = {}
-        for sensor in self.sensors:
-            data[sensor.name] = sensor.getJson()
+        sensor_data = Counter()
+        for key, value in self.sensors.items():
+            sensor_data += Counter(value.get_json())
 
-        return data
+        return sensor_data
 
     def run(self):
         while True:
             # time.sleep(self.interval_min*60)
             time.sleep(1) # for debugging
```

Comment:
Need to flip back to using the config interval. perhaps changing the config to set the frequency to 1sec